### ISSUE TITLE
Gbypas is broken in latest 5.5.3

### ISSUE THREAD

**Issue by fredbcode** [e2guardian/e2guardian#785]
There are two different cases: 

Sometimes gbypass is not generated at all (nothing in uri about BYPASS)

http://denypage/denied.php?DENIEDURL=https%3a%2f%2finstant%2dhack%2eto&IP=192.168.6.254&USER=test3&FILTERGROUP=groupe3&CATEGORIES=&EXTFLAGS=1344:I:icaph:defaultusermap&REASON=Site%20Interdit%20%3a%20Le%20mode%20liste%20blanche%20%28Blanket%20Block%29%20est%20actif%20et%20ce%20site%20n%60est%20pas%20dans%20la%20liste%20autoris%c3%a9e%2c%20blanche%20ou%20grise%2e%20

Or the uri is well formatted (with GBYPASS)

http://denypage/denied.php?DENIEDURL=https%3a%2f%2finstant%2dhack%2eto&IP=192.168.6.254&USER=test3&FILTERGROUP=groupe3&CATEGORIES=&GBYPASS=2F36996D9483123C74BECC5FC20F0A773557552880&EXTFLAGS=1344:I:icaph:defaultusermap&REASON=Site%20Interdit%20%3a%20Le%20mode%20liste%20blanche%20%28Blanket%20Block%29%20est%20actif%20et%20ce%20site%20n%60est%20pas%20dans%20la%20liste%20autoris%c3%a9e%2c%20blanche%20ou%20grise%2e%20

But we can't surf temporarily 

Just switching to previous binary (5.4.7.r) and all works well

**Comment by philipianpearce** [comment]
Hi Fred,

That is odd.   Are there any entries in access.log for these?

**Comment by fredbcode** [comment]
No nothing, unfortunately we are using e2 in docker (so without syslog) and I found no way to show debug logs in stdout

**Comment by philipianpearce** [comment]
The new debug logging in v5.5 allows you to output any debug type to stdout with config in e2guardian.conf.

E.g.  To output storyboard trace add:

storyboardtrace = on
set_storytrace = "stdout"

To debug icap:

debugformat = 4
debuglevel = "icap:stdout"

Regards, Philip

**Comment by philipianpearce** [comment]
I think I have found the problem - the bypass setting was not being read in - it somehow must have got deleted at some point.
Can you try the latest v5.5 and see if that fixes the problem?

**Comment by fredbcode** [comment]
Hi Philip
It's better GBYPASS is well generated now, but the domain still denied 

```
2023.08.31 09:28:45 test3 10.x.x.x https://lkjh.com/?GBYPASS=C2CC90F0BE744DF68ABA6759CCFF07E21693510112 *DENIED* Site Interdit : Le mode liste blanche (Blanket Block) est actif et ce site n`est pas dans la liste autorisée, blanche ou grise.  GET 0 0  3 403  10.253.33.61 groupe3 Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:102.0) Gecko/20100101 Firefox/102.0  - - 
```

**Comment by philipianpearce** [comment]
Confirm that Bypass code was broken in v5.5.
Have now reverted (mostly) bypass code to code from v5.4  and now appears to be working OK.
Also, fixed bug where bypass message was not appearing in access log.

Please check latest version of v5.5.

**Comment by fredbcode** [comment]
I confirm it works, thanks

**Comment by philipianpearce** [comment]
Also fixed in v5.6.dev.
